From: John Zupin <johnz@lunarg.com>
Date: Thu, 25 May 2023 14:26:10 -0600
Subject: patch: remove install objdump

---
 tools/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index a93f640..aa9e774 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -77,7 +77,7 @@ if (NOT ${SPIRV_SKIP_EXECUTABLES})
                       LIBS ${SPIRV_TOOLS_FULL_VISIBILITY})
     target_include_directories(spirv-objdump PRIVATE ${spirv-tools_SOURCE_DIR}
                                                      ${SPIRV_HEADER_INCLUDE_DIR})
-    set(SPIRV_INSTALL_TARGETS ${SPIRV_INSTALL_TARGETS} spirv-objdump)
+    #set(SPIRV_INSTALL_TARGETS ${SPIRV_INSTALL_TARGETS} spirv-objdump)
   endif()
 
   if(NOT (${CMAKE_SYSTEM_NAME} STREQUAL "iOS"))
